ext/opcache/ZendAccelerator.c

```diff
@@ -3010,10 +3010,15 @@ static int accel_post_startup(void)
 
 		zend_shared_alloc_lock();
 #ifdef HAVE_JIT
+		zend_long jit_flags = ZCG(accel_directives).jit_optimization_level * 1 +
```

Comment:
I think this should emit a notice or startup error if any of the individual flags is out of range. This is somewhat of a pre-existing issue, the jit didn't warn before.

For example, `ZCG(accel_directives).jit_optimization_level < 0 ||  ZCG(accel_directives).jit_optimization_level > 5` should emit a notice along the lines of `opcache.jit_optimization should be an integer between 0 and 5, but got %d`

(I'd agree that the opcache.jit bitmask would be better off as individual settings and this is an improvement, especially since there was previously no way to change just one of these)